### DISCUSSION TITLE
单双页页码

### DISCUSSION THREAD

**Discussion by W2Icon** [sjtug/SJTUThesis#916] [提问] [answered]
请问怎么设置单页页码排在每页右下角，双页页码排在每页左下角

**Comment by LogCreative** [comment, marked as answer]
导言区添加
```latex
\ExplSyntaxOn
\fancyfoot [ C ] {}
\fancyfoot [ LE, RO ] { \__sjtu_page:n { \__sjtu_thepage: } }
\ExplSyntaxOff
```

**Reply by W2Icon** [reply]
感谢解答！我在stjuthesis.cls进行了如下修改也达到想要的效果。
\bool_if:NTF \g__sjtu_twoside_bool
  {
    \fancyhead [ LO, RE ] { \l__sjtu_header_tl }
    \fancyhead [ LE ]     { \__sjtu_nouppercase:V \l__sjtu_leftmark_tl  }
    \fancyhead [ RO ]     { \__sjtu_nouppercase:V \l__sjtu_rightmark_tl }
    \fancyfoot [ LE, RO ] { \__sjtu_page:n { \__sjtu_thepage: } }
  }
  {
    \fancyhead [ L ] { \l__sjtu_header_tl }
    \fancyhead [ R ] { \__sjtu_nouppercase:V \l__sjtu_rightmark_tl }
  }
% \fancyfoot [ R ] { \__sjtu_page:n { \__sjtu_thepage: } }
